OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -1,31 +1,23 @@
 # Default values for iqserver.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-
+replicaCount: 1
+image:
+  repository: sonatype/nexus-iq-server
+  pullPolicy: IfNotPresent
+  
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server
-  imageTag: 1.85.0-01-ubi
-  imagePullPolicy: IfNotPresent
-  imagePullSecret: ""
+  hostname: iq-server.demo
+  imageName: sonatype/nexus-iq-server
+  imageTag: 1.90.0
   applicationPort: 8070
   adminPort: 8071
   memory: 1Gi
   # base 64 encoded license file with no line breaks
   licenseSecret: ""
-  # configYaml is the full text of the config.yml file that will be passed to IQ Server
-  configYaml:
-    server:
-      applicationConnectors:
-        - type: http
-          port: 8070
-      adminConnectors:
-        - type: http
-          port: 8071
-    createSampleData: true
-    sonatypeWork: /sonatype-work
-    # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
-    # licenseFile: /etc/nexus-iq-license/license_lic
+  # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
+  # licenseFile: /etc/nexus-iq-license/license_lic
```

Comment:
What's the motivation to bump the licenseFile up out of configYaml? Is it to match the licenseSecret config?

Reply:
Not sure I saw the connection. The config.yml section IS the config.yml that the app ends up using. My initial focus was on logging but want to come back and do some testing on the license stuff.

Reply:
the `licenseFile` is a part of `configYaml` so this comment should be moved back to that block